ios/Runner/AppDelegate.m

```diff
@@ -0,0 +1,28 @@
+//  AppDelegate.m
```

Comment:
This file should be removed.

The file has syntax errors and its not included in the iOS project.